<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSDlgLuminanceAndContrastSetting</class>
 <widget class="QDialog" name="CSDlgLuminanceAndContrastSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>130</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>130</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Luminance/Contrast Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label_Luminance">
       <property name="text">
        <string>Luminance</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Contrast">
       <property name="text">
        <string>Contrast</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSlider" name="horizontalSlider_Contrast">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSlider" name="horizontalSlider_Luminance">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_Luminance"/>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_Contrast"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
